lib/evmone/eof.hpp

```diff
@@ -192,7 +192,7 @@ enum class EOFValidationError
 
 enum class ContainerKind : uint8_t
 {
-    /// Container that uses RETURNCODE. Can be used by EOFCREATE/Creation transaction.
+    /// Container that uses RETURNCODE. Can be used by EOFCREATE/TXCREATE/Creation transaction.
```

Comment:
```suggestion
    /// Container that uses RETURNCODE. Can be used by EOFCREATE/TXCREATE.
```
The create transaction is on the way out?

Reply:
It is, not in this PR though.

Reply:
Removing it in https://github.com/ethereum/evmone/pull/1190